utils/change.js

```diff
@@ -21,7 +21,7 @@ function doChangeCalculations(row, previousRow, rowConfig, isDecennial) {
 }
 
 /*
- * Calculate change_sum, change_m, and change_significant
+ * Calculate change_sum, change_m, and change_reliable
```

Comment:
I know the other variable names in this comment are also out of date but shouldn't this say `changeReliable` or `change.reliable` to match up with the actual code? I think anywhere you see the underscore variables in these comments is because of stuff we refactored a while ago without updating the comments accordingly.

Reply:
Yep you're right, can change that. Didn't think about it too much at first because all the comments are outdated and needs a rework in subsequent PRs

Reply:
Yea makes sense. I'm cool with generally leaving the comments as-is here.